components/NavBar.js

```diff
@@ -0,0 +1,26 @@
+import styles from "./css/Header.module.css";
+
+const NavBar = () => {
+  return (
+      <nav className={styles.navbar}>
+        <h4 className={styles.brandlogo}>
+          <a href="/">
+            <img src="./icons/dinosaur.png" width="60" />
+          </a>
+          Evolution
+        </h4>
+        <ul>
+          <li>
+            <a href="https://github.com/SamSamskies/evolution" target="_blank">
+              Get Involved!
+            </a>
+          </li>
+          <li>
+            <a href="/about">About</a>
```

Comment:
Same here use the `Link` component for internal links. https://nextjs.org/docs/api-reference/next/link

Sorry, missed this on the first pass on this PR.

Reply:
This should prevent the full page refresh when clicking on the links.

Reply:
addressed on last commit